src/main/java/megamind/exception/InvalidCommandException.java

```diff
@@ -0,0 +1,16 @@
+package megamind.exception;
+
+/**
+ * Exception thrown when an invalid command is encountered.
+ */
+public class InvalidCommandException extends Exception {
+
```

Comment:
I like how you separated all the exceptions into different ones specific to the exception it pertains to.